api/src/pcapi/alembic/versions/20240102T141914_16ebdf83377e_add_elementary_student_level.py

```diff
@@ -0,0 +1,24 @@
+"""Add ECOLES_INNOVANTES_MARSEILLE_EN_GRAND_ELEMENTAIRE to student levels values"""
+from alembic import op
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "16ebdf83377e"
+down_revision = "ec0f52ada8fa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+        """
+        ALTER TYPE studentlevels
+        ADD VALUE IF NOT EXISTS 'ECOLES_INNOVANTES_MARSEILLE_EN_GRAND_ELEMENTAIRE';
```

Comment:
```suggestion
        ALTER TYPE studentlevels 
        RENAME VALUE 'ECOLES_INNOVANTES_MARSEILLE_EN_GRAND_PRIMAIRE' TO 'ECOLES_INNOVANTES_MARSEILLE_EN_GRAND_ELEMENTAIRE';
;
```
Est-ce que l'on ne devrait pas renommer la valeur, pour éviter de trainer une valeur `ECOLES_INNOVANTES_MARSEILLE_EN_GRAND_PRIMAIRE` inutile en testing ?